docs/nnUNet.md

```
@@ -0,0 +1,231 @@
+
+To run inference, run the command:
+
+```sh
+nnUNet_predict -i INPUT_FOLDER -o OUTPUT_FOLDER -t TASK_NAME_OR_ID -m CONFIGURATION
+```
+
+In this case, the `INPUT_FOLDER` of nnUNet is the `OUTPUT_DIRECTORY` of Med-ImageTools.# Preparing Data for nnUNet
+
+nnUNet repo can be found at: <https://github.com/MIC-DKFZ/nnUNet>
+
+## Processing DICOM Data with Med-ImageTools
```

Markdownlint

Multiple headings with the same content

(MD024, no-duplicate-heading)